src/pages/admin/AdminNavbar.js

```diff
@@ -1,86 +1,86 @@
-import React from 'react';
-import { NavHashLink } from 'react-router-hash-link';
+import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
-import { useAuthUser, useSignOut } from 'react-auth-kit';
+import { useSignOut } from 'react-auth-kit';
 import { OverlayTrigger, Popover, Badge } from 'react-bootstrap';
-import { jjcIcon, UserDropDownIcon, NotificationIcon } from '../../assets/index';
+import { BsBell } from 'react-icons/bs';
+import { CgProfile } from 'react-icons/cg';
+import { GoHome } from 'react-icons/go';
+import { FiLogOut } from 'react-icons/fi';
 import Avatar from '../../components/avatar/Avatar';
-import {BsBell} from 'react-icons/bs';
-import {CgProfile} from 'react-icons/cg';
-import {GoHome} from 'react-icons/go';
-import {FiLogOut} from 'react-icons/fi';
+import { jjcIcon, UserDropDownIcon } from '../../assets/index';
 import './AdminNavbar.scss';
+import GlobalContext from '../../store/GlobalContext';
 
 function AdminNavbar() {
-    const signOut = useSignOut();
-    const auth = useAuthUser();
+  const signOut = useSignOut();
+  const { globalState, setGlobalState } = useContext(GlobalContext);
```

Comment:
I think you don't need the setGlobalState function here, ideally setGlobalState should only be imported if you want to update the global state.

Reply:
Updated done